pytest_timeout.py

```diff
@@ -21,7 +21,7 @@
 __all__ = ("is_debugging", "Settings")
 SESSION_TIMEOUT_KEY = pytest.StashKey[float]()
 SESSION_EXPIRE_KEY = pytest.StashKey[float]()
-
+PYTEST_FAILURE_MESSAGE = "Timeout (>%ss) from pytest-timeout."
```

Comment:
```suggestion
PYTEST_FAILURE_TEMPLATE = "Timeout (>%ss) from pytest-timeout."
```
maybe?  It's a little confusing otherwise to me